CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ if (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
 endif()
 
 # Set required C++ standard
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Does this mean C++11 will no longer be supported?

Reply:
I think so, but I can not use lambda capture initializers, so we might have some unnecessary copies. Does the new approach look better ?